package.json

```diff
@@ -33,7 +33,7 @@
     "bootstrap-no-build-no-scripts": "lerna bootstrap --no-ci --force-local --ignore-scripts",
     "bootstrap-no-build-no-concurrency": "lerna bootstrap --no-ci --force-local --concurrency 1",
     "bootstrap-no-build-quiet": "lerna bootstrap --no-ci --force-local --loglevel=error",
-    "ci:bootstrap": "lerna bootstrap --no-ci --force-local --ignore-scripts && lerna run prepublish",
```

Comment:
this is while we're in here, but shouldn't the "not no scripts one" not ignore scripts?